packages/client/tests/functional/issues/22947-sqlite-conccurrent-upsert/_matrix.ts

```diff
@@ -0,0 +1,4 @@
+import { defineMatrix } from '../../_utils/defineMatrix'
+import { allProviders } from '../../_utils/providers'
+
+export default defineMatrix(() => [allProviders])
```

Comment:
Also rename this directory?

Reply:
I have the `optOut` property

```js
{
    optOut: {
      from: ['sqlserver', 'mongodb', 'postgresql', 'cockroachdb', 'mysql'],
      reason: 'Test is made for SQLite only',
    },
}
```

Reply:
Then let's change the matrix, because this is weird like this, right?